example/data-migration2-tf/main.tf

```diff
@@ -0,0 +1,42 @@
+terraform {
+  required_providers {
+    aws  = ">= 3.5.0"
+  }
+}
+
+provider "aws" {
+  region  = var.region
+  ignore_tags {
+    key_prefixes = ["gsfc-ngap"]
+  }
+}
+
+provider "aws" {
+  alias   = "usw2"
+  region  = "us-west-2"
+  profile = var.aws_profile
+}
+
+data "terraform_remote_state" "data_persistence" {
+  backend   = "s3"
+  config    = var.data_persistence_remote_state_config
+  workspace = terraform.workspace
+}
+
+module "data_migration2" {
+  source = "../../lambdas/data-migration2"
```

Comment:
(and I realize this is following the pattern we have for the data-migration1 deployment)

Reply:
Oh good points!

So re: separate deployments for the data-migration lambdas: it seems like it would simplify things for users if we didn't do that and just bundled 'em with example/cumulus-tf. I don't know if we'd want to spin up a container with every cumulus example deploy though so they would still need to deploy `data-migration-ecs` separately I think. But that's an easier lift.

I'm looking for downsides here. I guess these Lambdas are ephemeral and for current users only. If you're deploying cumulus for the first time post-RDS parallel writes, you won't use these data-migration Lambdas. But so long as we don't spin up a container, it shouldn't cost anything so 🤷.

I'm game to bundle these with example/cumulus-tf unless there's something I'm not seeing.

Reply:
Resolved. Thanks for pairing on this!